src/app/modules/group/pages/group-administration/group-administration.component.ts

```diff
@@ -10,16 +10,10 @@ import { canCurrentUserManageGroup } from '../../helpers/group-management';
 })
 export class GroupAdministrationComponent {
 
-  group: Group
-  isAllowed: boolean
+  group$ = this.groupTabService.group$.pipe(map((g)=>withManagementAdditions(g)));
```

Comment:
```suggestion
  group$ = this.groupTabService.group$.pipe(map((g) => withManagementAdditions(g)));
```

Reply:
" smadbe marked this conversation as resolved. "  -> note that GH did it for me when accepting the changes (as it violates my own rules ;-))